src/apps/relay/mainrelay.c

```diff
@@ -2328,6 +2329,10 @@ static void set_option(int c, char *value) {
       turn_params.rest_api_separator = *value;
     }
     break;
+  case UDP_NOTIFIER_OPT:
+    turn_params.udp_notifier = 1;
```

Comment:
please use the C99 "stdbool" header, and the true/false keywords (sometimes implemented as macros by C99) over '1' and '0' and 'int'.